server/js/sockets.js

```diff
@@ -149,6 +150,14 @@ sockets.init = function socketInit(server) {
             console.error(`Unhandled topic - ${topic}`);
             break;
         }
+      } else if (
+        topicString.slice(0, -1).join('/') === '/v3/camera/recording/status'
+      ) {
+        // send mqtt payload on corresponding channel
```

Comment:
Same here.

Reply:
Done
130e0c0

Not sure why it wasn't picked up either